emily/handler/src/api/models/withdrawal/mod.rs

```diff
@@ -56,6 +56,8 @@ pub struct Withdrawal {
     /// Details about the on chain artifacts that fulfilled the withdrawal.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub fulfillment: Option<Fulfillment>,
+    /// The txid of the stacks transaction that generated this event.
```

Comment:
I would change this (and the others) to be a little more specific.
```suggestion
    /// The hex encoded txid of the stacks transaction that generated this event.
```